.github/workflows/integration_test.yaml

```diff
@@ -141,6 +141,10 @@ env:
   REGISTRY: ghcr.io
   OWNER: ${{ github.repository_owner }}
 
+concurrency:
+  group: ${{ github.workflow }}-integration-tests-${{ github.ref }}
```

Comment:
I think maybe we should add a `operator-workflows` prefix to the concurrency group name to prevent potential collisions with other workflows.

Reply:
Good point, added, thanks